spec.html

```diff
@@ -35357,7 +35363,7 @@ <h1>
             1. Assert: _c_ is a Continuation.
             1. Let _Input_ be _x_'s _input_.
             1. Let _e_ be _x_'s _endIndex_.
-            1. If _e_ = 0, or if _rer_.[[Multiline]] is *true* and the character _Input_[_e_ - 1] is one of |LineTerminator|, then
+            1. If _e_ = 0 or _rer_.[[Multiline]] is *true* and the character _Input_[_e_ - 1] is matched by |LineTerminator|, then
```

Comment:
```suggestion
            1. If _e_ = 0, or if _rer_.[[Multiline]] is *true* and the character _Input_[_e_ - 1] is matched by |LineTerminator|, then
```
In this case "or if" isn't just easier to read, it's also less ambiguous (English does not have precedence rules for "or" vs "and", so "if x or y and z" is always ambiguous).

Ditto several places below.